### DISCUSSION TITLE
Unity에서 (Play)시 카메라 화면 회전에 관한 문제

### DISCUSSION THREAD

**Discussion by MAX-Jeon** [naverz/zepeto-studio-kor#735] [5. Q&A]
![image](https://user-images.githubusercontent.com/102704503/167969205-7ca607fe-5cda-45aa-a1b9-a89de4ea67cf.png)

naverz/ zepeto - multiplay - example - main
샘플을 그대로 사용하였습니다.

**위 화면에 빨간색으로 표시한 부분을 마우스로 드래그해도 카메라가 회전 되지 않고 있습니다.
다른 방법이 있는 걸까요 ???

몇 차례 유니티도 재설치하고 프로젝트를 다시 생성해봐도 똑같은 결과가 나오네요.
추가적인 설정이 필요하거나 혹시 같은 문제를 해결하신 분이 있다면 공유 부탁드리겠습니다.**

"
- 유니티 시작 시 Input System 경고창 Yes 버튼
- Project Settings -> player -> Active Input Handling : Both
- Event System - Replace with InputSystemUIInputModule 버튼 클릭
- zepeto Login
- open world setting - 'Horizontal'
- play 진행
"
![image](https://user-images.githubusercontent.com/102704503/167970437-ccea72a9-7d19-450b-909e-61c5c48f6021.png)

**Comment by pumfit** [comment]
게임뷰에 있는 해상도가 Free Aspect로 되어있어서 직접 기기에 맞는 해상도로 설정하시면 될 것 같습니다!
<img width="1005" alt="image" src="https://user-images.githubusercontent.com/46295539/167984785-aa419f2b-b79b-49ea-97ce-f6f0f9cf7335.png">
